lib/adagrams.rb

```diff
@@ -0,0 +1,70 @@
+# Wave 1 [Question 1 - Creating Data Structure]
+
+def draw_letters
+    pool_of_letters = [
+    "A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", "C", "D", "D", "D", "D", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "J", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N", "O", "O", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R", "R", "R", "R", "R", "S", "S", "S", "S", "T", "T", "T", "T", "T", "T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", "Y", "Z"
+]
+    my_letters = []
+    random_letter = pool_of_letters.shuffle
+    10.times do
+        my_letters.push(random_letter[-1])
+        random_letter.pop
+    end
+    return my_letters
+end 
+
+def uses_available_letters?(input, letters_in_hand)
+    word = input.upcase.split(//)
```

Comment:
I just remembered that I was supposed to get in touch with Cloudy so that her VS Code indented with 2 spaces instead of 4 :)